app/scripts/directives/deploymentDonut.js

```diff
@@ -53,6 +56,23 @@ angular.module('openshiftConsole')
         $scope.$watchGroup(['limitRanges', 'hpa', 'project'], updateHPAWarnings);
         $scope.$watch('rc.spec.template.spec.containers', updateHPAWarnings, true);
 
+        var updateQuotaWarning = function() {
+          if (_.get($scope.rc, 'spec.replicas', 0) > _.get($scope.rc, 'status.replicas', 0)) {
```

Comment:
It looks like `spec.replicas` is required for replica sets and defaults to 1 for pet sets.